src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -6,9 +6,142 @@ service: Xata
 stub: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
 
-[Xata](https://xata.io) is a **Serverless Data Platform** that combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+[Xata](https://xata.io) is a **Serverless Data Platform** because it combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+## Adding a database with Xata
+
+### Prerequisites
+
+- A [Xata](https://app.xata.io/signin) account with a created database (you can use the sample database from the Web UI).
+- An Access Token (`XATA_TOKEN_API`).
+
+### Environment configuration
+
+The quickest way to integrate is via the Xata CLI
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn global install @xata.io/cli@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+The package will expose the `xata` command to your `$PATH` and now it's possible to initialize the integration:
+
+```shell
+xata init
+```
+
+:::note
+You can just execute the binary with `npx`, `pnpm dlx`, or `yarn dlx` if preferred.
+:::
+
+Once you follow the prompt, you should have a `.env` file in the root of your project with the following variables defined:
+
+```ini title=".env"
+XATA_API_KEY=hash_key
+
+# Xata branch that will be used
+# if there's not a xata branch with
+# the same name as your git branch
+XATA_FALLBACK_BRANCH=main
+```
+
+To have IntelliSense for your environment variables, edit or create the file `env.d.ts` in your `/src` directory to make your environment variables type-safe:
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly XATA_API_KEY: string;
+  readonly XATA_FALLBACK_BRANCH?: string;
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv;
+}
+```
+
+:::tip
+Read more about [environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
+:::
+
+And a `.xatarc` file with a `databaseURL`. Though it can be extended to have more definitions, such as the output path for your codegen, e.g.:
+
+```json title=".xatarc"
+{
+  "databaseURL": "https://{{ Xata DB URL }}",
+  "codegen": {
+    "output": "src/xata.codegen.ts"
+  }
+}
+```
+
+Additionally, the prompt may have assisted you on creating an instance of the SDK (Software Development Kit) and, if you chose TypeScript, you also have the types generated from your schema. Notice `@xata.io/client` was added to your `package.json` as well.
```

Comment:
I believe here the preposition should be "in" instead.
```suggestion
Additionally, the prompt may have assisted you in creating an instance of the SDK (Software Development Kit) and, if you chose TypeScript, you also have the types generated from your schema. Notice `@xata.io/client` was added to your `package.json` as well.
```

Reply:
Great catch!! Thanks!